commands/id.js

```diff
@@ -0,0 +1,46 @@
+const command = {};
+
+const util = require('../lib/util');
+
+command.description = 'Find a user\'s ID';
+
+command.usage = '<name|name#1234>';
+
+command.comment = 'This will only find the user if he is in a common guild with the bot';
+
+command.names = ['id'];
+
+command.execute = async (message, args, database, bot) => {
+  let id;
```

Comment:
```suggestion
  let response;
```
Rename id to response, because it can also contain error messages.